### PR TITLE
Merge master to dev for 1.6.0 beta.1

### PR DESCRIPTION
**Pull request by ddimatos**
##### SUMMARY
Part of the release for 1.6.0-beta.1 is to merge all the updated meta data back to dev for releases to build onto. This is for issue #606

### CODE REVIEW

**Comment by ddimatos** [comment]
Log `job/zosAnsibleAlternate/63/console`
image

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,51 @@ ibm.ibm_zos_core Release Notes
...
+-------------
+
+- zos_volume_init - Introduces new module to handle volume (or minidisk) initialization. (https://github.com/ansible-collections/ibm_zos_core/pull/654)
+
+Minor Changes
+-------------
+
+- Updated the text converter import from "from ansible.module_utils._text" to "from ansible.module_utils.common.text.converters" to remove warning".. warn Use ansible.module_utils.common.text.converters instead.". (https://github.com/ansible-collections/ibm_zos_core/pull/602)
+- module_utils - job.py utility did not support positional wiled card placement, this enhancement uses `fnmatch` logic to support wild cards.
+- zos_copy - Fixed a bug where the module would change the mode for a directory when copying into it the contents of another. (https://github.com/ansible-collections/ibm_zos_core/pull/723)
```
Comment (fernandofloresg) [inline CHANGELOG.rst:30]: There is a bug fix inside the minor changes tag. I changed it for release v1.4.1 but apparently failed to communicate with the team, no need to run the pipeline again

Comment (ddimatos) [reply CHANGELOG.rst:30]: OK, maybe we can add it after this PR?

File: meta/ibm_zos_core_meta.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: ibm_zos_core
-version: "1.5.0"
+version: meta/ibm_zos_core_meta.yml6.0-beta.1"
```
Comment (fernandofloresg) [inline meta/ibm_zos_core_meta.yml:2]: Why this version name is meta/ibm_zos_core_meta.yml6.0-beta.1? looks like a typo

**Review by fernandofloresg — CHANGES_REQUESTED** [review]
I think we only need to change the version name in the meta

**Review by fernandofloresg — APPROVED** [review]

**Review by ketankelkar — APPROVED** [review]
lgtm after seeing resolutions to comments by @fernandofloresg

**Pull request merged**